articles/multifactor-authentication/developer/step-up-with-acr.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
Would be useful to add a note on the MFA documents of each quickstarts: https://auth0.com/docs/quickstart/spa/vanillajs/09-mfa.